conf/pyramid/config.jsonnet

```diff
@@ -8,10 +8,10 @@ local sections = import 'sections.libsonnet';
 
 {
   // Output INI files
-  'demo.ini': std.manifestIni(rc_and_demo_ini),
-  'demo_frontend.ini': std.manifestIni(rc_and_demo_frontend_ini),
-  'rc.ini': std.manifestIni(rc_and_demo_ini),
-  'rc_frontend.ini': std.manifestIni(rc_and_demo_frontend_ini),
+  'demo.ini': std.manifestIni(demo_ini),
+  'demo_frontend.ini': std.manifestIni(demo_frontend_ini),
```

Comment:
I suppose technically demo_frontend and rc_frontend are identical but I'm not sure we ever create demo_frontend anyway, I guess for a demo cluster.